ngraph/frontend/onnx_import/include/onnx_import/core/node.hpp

```diff
@@ -96,6 +98,114 @@ namespace ngraph
+
+        template <>
+        ONNX_IMPORTER_API std::vector<std::int64_t>
+            Node::get_attribute_value(const std::string& name,
+                                      std::vector<std::int64_t> default_value) const;
+
+        template <>
+        ONNX_IMPORTER_API std::vector<std::size_t>
+            Node::get_attribute_value(const std::string& name,
+                                      std::vector<std::size_t> default_value) const;
+
+        template <>
+        ONNX_IMPORTER_API std::vector<std::string>
+            Node::get_attribute_value(const std::string& name,
+                                      std::vector<std::string> default_value) const;
+
+        template <>
+        ONNX_IMPORTER_API std::vector<Tensor>
+            Node::get_attribute_value(const std::string& name,
+                                      std::vector<Tensor> default_value) const;
+
+        template <>
+        ONNX_IMPORTER_API std::vector<Graph>
+            Node::get_attribute_value(const std::string& name,
+                                      std::vector<Graph> default_value) const;
+
+        template <>
+        ONNX_IMPORTER_API float Node::get_attribute_value(const std::string& name) const;
+
+        template <>
+        ONNX_IMPORTER_API double Node::get_attribute_value(const std::string& name) const;
+
+        template <>
+        ONNX_IMPORTER_API std::int64_t Node::get_attribute_value(const std::string& name) const;
+
+        template <>
+        ONNX_IMPORTER_API std::size_t Node::get_attribute_value(const std::string& name) const;
+
+        template <>
+        ONNX_IMPORTER_API std::string Node::get_attribute_value(const std::string& name) const;
+
+        template <>
+        ONNX_IMPORTER_API Tensor Node::get_attribute_value(const std::string& name) const;
+
+        template <>
+        ONNX_IMPORTER_API Subgraph Node::get_attribute_value(const std::string& name) const;
+
+        template <>
+        ONNX_IMPORTER_API std::vector<float>
+            Node::get_attribute_value(const std::string& name) const;
+
+        template <>
+        ONNX_IMPORTER_API std::vector<double>
+            Node::get_attribute_value(const std::string& name) const;
+
+        template <>
+        ONNX_IMPORTER_API std::vector<std::int64_t>
+            Node::get_attribute_value(const std::string& name) const;
+
+        template <>
+        ONNX_IMPORTER_API std::vector<std::size_t>
+            Node::get_attribute_value(const std::string& name) const;
+
+        template <>
+        ONNX_IMPORTER_API std::vector<std::string>
+            Node::get_attribute_value(const std::string& name) const;
+
+        template <>
+        ONNX_IMPORTER_API std::vector<Tensor>
+            Node::get_attribute_value(const std::string& name) const;
+
+        template <>
+        ONNX_IMPORTER_API std::vector<Graph>
+            Node::get_attribute_value(const std::string& name) const;
```

Comment:
Should we add new nGraph tests? For this functionality?

Reply:
those functions are heavily used in ngraph/frontend/onnx_import/src/op and we have this code covered in our tests.